djangocms_versioning/cms_config.py

```diff
@@ -147,11 +147,11 @@ def emit_page_content_change(version):
     if installed. It is used for re-indexing version state info
     """
     try:
```

Comment:
Perhaps do
```python
try:
    from djangocms_internalsearch.signals import content_object_state_change
except ImportError:
    def emit_page_content_change(version):
        pass
else:
    def emit_page_content_change(version):
        # implementation
        ...
```

Reply:
one million ways to skin a cat, isn't your previous suggestion just fine?

Reply:
I honestly find that easier to ready. Does this have a performance advantage?

Reply:
Doesn't it attempt to import every time the function is called?